triage-process.md

```diff
@@ -0,0 +1,60 @@
+    1. Workability Classification (also below are label names):
+        1. feature-request
+        1. enhancement
+        1. bug
+        1. wont-fix
+1. Workable Issues should be prioritized and Labeled as one of 3 priorities
+    * P0 = Critical (Drop everything and work on it.)
+    * P1 = Important (Work on it if no P0 are present. Use judgment to prioritize items within a list of P1s)
+    * P2 = Low (Work on it when no P1 are left or some one from community wants to contribute)
+1. Prioritized Issues (and unworked) issues should be labeled as ‘Ready for Work’. This should signal to any contributor willing to help that they can work on this issue.
+1. Once a contributor is working on this issue, the issue should be:
+    1. Assigned to them
+    1. Labeled as ‘In Progress’
+1. Once work is complete and issue has been resolved it should be labeled as ‘Completed’ and Issue should be closed. 
+
+
+### Issue Stages
+Stage | Meaning | Action/Label
+------|---------|-------------
+New | Ticket is ready for triage. | Issue should be labeled ‘New’ 
+Ready for Work | Ticket triaging has been completed and is up for grabs to be picked up to be worked on by a contributor. | Ticket is labeled ‘Ready for work’. Label ‘New’ should be removed when moving to this stage.
+In Progress | Ticket is actively being worked upon by a contributor. | Contributor who has picked up this ticket shall assign the ticket to themselves.
+In Review | Implementation work related to the ticket has been completed. Work related to a Review and/or any changes requested as part of the review  is in progress.  | Issues with a PR associated with it will be considered as in ‘In Review’
+Completed | Work for this issue has been completed and reasonably tested. | No label is needed. Issue is simply closed.
+Blocked | Issue is blocked for one reason or another. Typically where the delay or waiting period is more than 5 days. | Ticket should be labelled ‘Blocked’
+Won't Fix | Issue is not workable.  | Issue should have comment indicating why its not fixable. 
+
+
+### Roles
+* __Triager__: 
+    * Person who is triaging the issue by determining its workability. This person is responsible for getting the ticket to one of two labels - 1) Ready for Work 2) Cancelled. They are responsible for triaging  by working with the OP to get additional information as needed and analyzing the issue and adding relevant details/information/guidance that would be helpful to the resolution of the issue.
+
+* __OP__: 
+    * Original Poster. This is the person who has opened or posted the issue.
+
+* __Contributor__:
+    * Person(s) that are actually doing the work related to the ticket. Once work is done they work with the Reviewer and the Contributing Reviewers to get feedback implemented and complete the work. They  are responsible for making sure issue status label is up to date (In Progress, In Review, Completed, Blocked, Cancelled)
```

Comment:
Nit: Add comma i.e. `Once work is done, they ...`

Reply:
done